src/common/version.h

```diff
@@ -16,9 +16,10 @@
 #define CEPH_COMMON_VERSION_H
 
 #include <string>
+#include "common/ceph_context.h"
 
 // Return a string describing the Ceph version
-const char *ceph_version_to_str(void);
+const char *ceph_version_to_str(CephContext *cct);
```

Comment:
this function is also used by crimson. you might want to update `src/crimson/admin/admin_socket.cc` to pass `nullptr` to `ceph_version_to_str()`.